pod/perlgov.pod

```diff
@@ -0,0 +1,531 @@
+
+=item *
+
+Perl conferences and events, except those organized directly by Core Team
+
+=item *
+
+Perl-related intellectual property legally owned by third-parties, except as
+allowed by applicable licenses or agreements.
+
+=back
+
+=head3 Membership
+
+The initial Core Team members will be specified when this document is
+first ratified.
+
+Any Core Team member may nominate someone to be added to the Core Team by
+sending the nomination to the Steering Council.  The Steering Council must
+approve or reject the nomination.  If approved, the Steering Council will
+organize a Membership Change Vote to ratify the addition.
+
+Core Team members should demonstrate:
+
+=over
+
+=item *
+
+A solid track record of being constructive and helpful
+
+=item *
+
+Significant contributions to the project's goals, in any form
+
+=item *
+
+Willingness to dedicate some time to improving Perl
+
+=back
+
+Contributions are not limited to code. Here is an incomplete list of areas
+where contributions may be considered for joining the Core Team:
+
+=over
+
+=item *
+
+Working on community management and outreach
+
+=item *
+
+Providing support on mailing lists, IRC, or other forums
+
+=item *
+
+Triaging tickets
+
+=item *
+
+Writing patches (code, docs, or tests)
+
+=item *
+
+Reviewing patches (code, docs, or tests)
+
+=item *
+
+Participating in design discussions
+
+=item *
+
+Providing expertise in a particular domain (security, i18n, etc.)
+
+=item *
+
+Managing Perl infrastructure (websites, CI, documentation, etc.)
+
+=item *
+
+Maintaining significant projects in the Perl ecosystem
+
+=item *
+
+Creating visual designs
+
+=back
+
+Core Team membership acknowledges sustained and valuable efforts that align
+well with the philosophy and the goals of the Perl project.
+
+Core Team members are expected to act as role models for the community and
+custodians of the project, on behalf of the community and all those who rely
+on Perl.
+
+=head3 Term
+
+Core Team members serve until they are removed.
+
+=head3 Removal
+
+Core Team Members may resign their position at any time.
+
+In exceptional circumstances, it may be necessary to remove someone from the
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+=head3 Inactivity
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves "inactive". Inactive members do not nominate or vote.  Inactive
+members may declare themselves active at any time, except when a vote has been
+proposed and is not concluded.  Eligibility to nominate or vote will be
```

Comment:
This seems plenty good enough as-is. I believe the spirit of this would be better phrased as "Inactive
members may declare themselves active at any time, but will not participate in votes which began while they were inactive." This or similar wording would also allow re-activation even during periods of concurrent or long-running votes.

Reply:
I take your point, but believe this will be exceptionally rare in practice.  If it does come up, I will propose a change such as this.  Thanks.